docs/source/development/documentation_cookbook.rst

```diff
@@ -0,0 +1,18 @@
+.. _core-development:documentation-cookbook:
+
+***********************
+Documentation cookbook
+***********************
+
+
+.. _core-development:orm:sphinxcheat:
+
+Sphinx cheatsheet
+==================
+
+
+.. _core-development:orm:visuals:
+
+Visual aides
```

Comment:
@ramirezfranciscof I thought we decided to call this `ASCII graphs`?

Reply:
Ah, sorry, I didn't remember this. Changed!

Reply:
(I thought in the doc that this was a description, he-he)